sci-libs/openlibm/openlibm-0.7.2.ebuild

```diff
@@ -1,23 +1,29 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit eutils toolchain-funcs fortran-2
+inherit toolchain-funcs
 
 DESCRIPTION="High quality system independent, open source libm"
 HOMEPAGE="https://github.com/JuliaLang/openlibm"
-SRC_URI="https://github.com/JuliaLang/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/JuliaMath/openlibm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="public-domain MIT ISC BSD-2 LGPL-2.1+"
 SLOT="0/${PV}.0"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 
 IUSE="static-libs"
 
+PATCHES=( "${FILESDIR}"/${PN}-0.7.2-make_inc.patch )
+
+pkg_setup() {
+	tc-export CC CXX FC AR LD
+}
+
 src_install() {
 	emake DESTDIR="${D}" prefix="${EPREFIX}/usr" \
 		libdir="${EPREFIX}/usr/$(get_libdir)" install
-	use static-libs || rm "${D}/${EPREFIX}/usr/$(get_libdir)/libopenlibm.a" || die "rm failed"
+	use static-libs || rm -f "${ED}/usr/$(get_libdir)/libopenlibm.a" || die "rm failed"
```

Comment:
remove static-libs, and preferably from the build system

Reply:
Hoping to upstream this patch, so we don't have to maintain.
They won't accept it we remove static building.
I think this is an okay compromise for now.